WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -94,6 +95,9 @@ protected SendControlViewModel(Wallet wallet, string title)
 			SetFeeTargetLimits();
 			FeeTarget = Global.UiConfig.FeeTarget;
 			FeeDisplayFormat = (FeeDisplayFormat)(Enum.ToObject(typeof(FeeDisplayFormat), Global.UiConfig.FeeDisplayFormat) ?? FeeDisplayFormat.SatoshiPerByte);
+			_transactionFeeInfo = this.WhenAnyValue(x => x.UsdFee, x => x.UsdExchangeRate, x => x.FeePercentage, x => x.FeeRate, x => x.EstimatedBtcFee,
+				(a, b, c, d, e) => new TransactionFeeInfo(a, b, c, d, e)
+			).ToProperty(this, x => x.TransactionFeeInfo, scheduler: RxApp.MainThreadScheduler);
```

Comment:
How about this to make it look better:
```suggestion
			_transactionFeeInfo = this.WhenAnyValue(
				x => x.UsdFee,
				x => x.UsdExchangeRate,
				x => x.FeePercentage,
				x => x.FeeRate,
				x => x.EstimatedBtcFee,
				(a, b, c, d, e) => new TransactionFeeInfo(a, b, c, d, e))
				.ToProperty(this, x => x.TransactionFeeInfo, scheduler: RxApp.MainThreadScheduler);
```